addons/crm_addons/models/res_partner.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    iap_enrich_info = fields.Text('IAP Enrich Info', help='Stores additional info retrieved from IAP in JSON')
```

Comment:
We decided not to do store extra iap infor to avoid bloating the partner model, and instead post a message. Why should it be different here ?

Reply:
That means for sure calling the IAP to enrich every time we need the company info as displayed in the app (like Industry, Employes, Year Founded, Revenue, etc.).

Then it was requested in the specs specifically by @awa-odoo : "Let's store the result of the enrichment (json data) on a new field on the res.partner model (iap_enrich_info ?)".

Then I discussed it with him on Discord to know whether to store the whole thing or just a selection of the fields, and he said to store the whole thing, he mentioned that he even had this conversation with you.